### PR TITLE
Feat/hash

### PR DESCRIPTION
**Pull request by eveleighoj**
A change to add hashing in as part of the process to attempt to reduce the amount of EFS files being added

### CODE REVIEW

File: tests/acceptance/test_sqlite_sync.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        conn = boto3.client("s3", region_name="us-east-1")
+        yield conn
+
+@pytest.fixture
+def testing_bucket(s3_client):
+    s3_client.create_bucket(Bucket='testing_bucket')
+    return 'testing_bucket'
+
+
+# @pytest.fixture
```
Comment (cpcundill) [inline tests/acceptance/test_sqlite_sync.py:30]: Probably worth removing?

File: tests/data/central-activities-zone.sqlite3.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
```
Comment (cpcundill) [inline tests/data/central-activities-zone.sqlite3.json:1]: Do we have a use case for the fields other than the hash in this JSON document?

File: task/utils.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import subprocess
+
+def generate_sqlite_hash(sqlite_path):
```
Comment (cpcundill) [inline task/utils.py:3]: Pretty neat really given that we had to call a separate shell process to obtain the sqlite hash

File: tests/data/central-activities-zone.sqlite3.json (hunk 2 above)
Comment (eveleighoj) [reply tests/data/central-activities-zone.sqlite3.json:1]: Not at the moment but I figure if we want to change tactics and include other things in the future then we can. Python dicts are also to store as json etc.

File: tests/acceptance/test_sqlite_sync.py (hunk 1 above)
Comment (eveleighoj) [reply tests/acceptance/test_sqlite_sync.py]: done

**Pull request merged**